### PR TITLE
Feature/four 10104

### PR DESCRIPTION
**Pull request by pmoskrojas**
ci:next

## Issue & Reproduction Steps
Describe the issue this ticket solves and describe how to reproduce the issue (please attach any fixtures used to reproduce the issue).

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
- Link to any related FOUR tickets, PRDs, or packages

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:next

### CODE REVIEW

**Review by gustavobascope — APPROVED** [review]
approved

**Review by boliviacoca — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
         "vue-deepset": "^0.6.3",
         "vue-events": "^3.1.0",
         "vue-monaco": "^1.2.1",
+        "vue-pagination-2": "^3.1.0",
```
Comment (nolanpro) [inline package.json:96]: @pmoskrojas is this being used anywhere? I don't see `import Pagination from 'vue-pagination-2';`

File: resources/js/notifications/components/NotificationsList.vue [hunk 2]
```diff
@@ -1,167 +1,252 @@
...
+          <notification-user
+            :notification="props.rowData"
+          />
+        </template>
+
+        <template
+          slot="subject"
+          slot-scope="props"
+        >
+          <a :href="props.rowData.url">
```
Comment (nolanpro) [inline resources/js/notifications/components/NotificationsList.vue:49]: @pmoskrojas Can you change this to a `@click` event and `cursor: pointer;` like resources/js/notifications/components/notification-item.vue does so we don't change the color?

Also, comments are not linking for some reason but they do link in the bell icon popover

File: resources/js/notifications/components/NotificationsList.vue [hunk 3]
```diff
@@ -1,167 +1,252 @@
...
+      if (this.cancelToken) {
+        this.cancelToken();
+        this.cancelToken = null;
+      }
+      const { CancelToken } = ProcessMaker.apiClient;
+
+      // Load from your API client (adjust the API endpoint and parameters as needed)
+      ProcessMaker.apiClient
+        .get(
+          `notifications?page=${
```
Comment (nolanpro) [inline resources/js/notifications/components/NotificationsList.vue:211]: @pmoskrojas All the rows with users have "Unknown User" I think you need to add `include=user` to the query to get the additional user info in the request

**Review by nolanpro — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (pmoskrojas) [reply package.json:96]: This is necessary for ./resources/js/components/common/Pagination.vue and other files which use Bootstrap too like ./resources/js/components/common/mixins/datatable.js. Generally for improve performance and current behavior.

**Review by nolanpro — APPROVED** [review]
Looks good

**Pull request merged**